iron-a11y-keys-behavior.html

```diff
@@ -411,14 +374,21 @@
           event.stopPropagation();
         }
 
-        keyBindings.forEach(function(keyBinding) {
-          var keyCombo = keyBinding[0];
-          var handlerName = keyBinding[1];
+        // if event has been already prevented, don't do anything
+        if (event.defaultPrevented) {
```

Comment:
I had the same doubt, but then checking the previous implementation, we weren't triggering any event if the original event was prevented. So over here i kept the same functionality